src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -276,6 +276,7 @@
   <xsd:complexType name="bind" mixed="true">
     <xsd:choice maxOccurs="unbounded">
       <xsd:element name="bind" type="argument" minOccurs="0" maxOccurs="unbounded" />
+      <xsd:element name="argument" type="argument" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
On closer review, I feel like I mislead you: maybe we can just set the type to "bind" on the line above and remove this one.
Yes, that would forbid XML files that use `<argument>` nested inside `<bind>`. But these were ignored, isn't it? In that case, I would definitely be fine "breaking" those files, proved they are really misleading, this broken also at the moment.

Reply:
Got it thanks! I updated the PR to add a new "bind_argument" complex type.

Reply:
Thanks, that works perfectly.